### ISSUE TITLE
Tabs component contains role="listbox", which confuse JAWS user

### ISSUE THREAD

**Issue by zhenwan** [carbon-design-system/carbon#6238]
> Automated testing tool and ruleset
JAWS
> Assistive technology used to verify
JAWS 

## Detailed description
role= 'listbox' is not correct word for this role, use Tablist or a Tab group instead of listbox to make it clear

> What version of the Carbon Design System are you using?
		"carbon-components": "^10.10.1",
		"carbon-components-react": "^7.10.1"

> What WCAG 2.1 checkpoint does the issue violate?
4.1.2 Name, Role, Value

The role="listbox" is in Tabs.js 
![image](https://user-images.githubusercontent.com/14806866/84207705-9c664680-aa77-11ea-95a2-a5bf4ec3bfe9.png)

**Comment by joshblack** [comment]
Hey there @zhenwan! 👋  Definitely agreed, we have an open issue for removing the listbox here: https://github.com/carbon-design-system/carbon/issues/4758

Going to close this out in preference of that issue 😄
